scripts/provisioning/s3_start

```diff
@@ -49,6 +57,11 @@ def main():
   try:
     if args.service == 's3server':
       # Add action and remove pass
+      if not args.index:
+        raise Exception("provide instance id")
+      fid_path = "/opt/seagate/cortx/s3/s3server-" + args.index
+      Fid = get_fid(fid_path)
+      os.system(f"sh /opt/seagate/cortx/s3/s3startsystem.sh {Fid} {fid_path} {s3config_path}")
```

Comment:
in K8 env check if entry point script has to exit or wait as process is running. os.system() will wait

Reply:
in case of s3 start entry script ,entry script completes it;s excecution,as start command s3server dont be in startup state always,but in case of auth and bg starting these binaries wont exit entry script.

Reply:
Discussed.  Need changes in s3server (cmdline arg to disable daemon mode).

Reply:
Impl. done - test pending.